CHANGELOG.md

```diff
@@ -3,6 +3,11 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/) and [Keep a changelog](https://github.com/olivierlacan/keep-a-changelog).
 
 ## [Unreleased](https://github.com/idealista/nginx_role/tree/develop)
+### Added
+- *[#101](https://github.com/idealista/nginx_role/issues/101) Debian 11 support* @sorobon
+### Removed
+- Debian 8 (jessie) support
```

Comment:
We should also link "[#101]" issue I guess, or at least the PR number (102)

Reply:
done